docs/_running-tests/automation.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: Running Tests in Automation
```

Comment:
If we want to suggest `wpt run` as the easiest approach for newcomers, I think "in Automation" gives the wrong impression. It sounds more like information about setting up CI systems.

I feel it would be better to put the first two paragraphs here should become `_running-tests/index.md`, followed by links to two subpages: one for the "Writing Your Own Runner" section, and one for setting up `./wpt serve`.

Does `./wpt run` need the hosts file setup as well? I suspect it isn't necessary for Gecko/Servo, but maybe it is for Chromium/WebKit? In that case, we need to find a way to weasel that into the front page too.

Reply:
I think this is the fundamental question that needs to be addressed here: how do we expect tests to be run in the normal individual user case?

Reply:
Re the "fundamental question", my vote goes that `wpt run` should be the default, recommended way to run tests, given how much setup we do in `wpt run` (not just dependencies, but also browser configurations).

Reply:
`wpt run` is less convenient when you want to attach a JS debugger, for example, though, which isn't that unusual of a case when writing tests.

Reply:
That sounds like a possible feature request rather than a fundamental limitation; we could support js debuggers like we support system-lvel debuggers (i.e. we add a --js-debug option that causes the browser to start with the debugger open and disables timeouts).